admin/database/postgres/postgres.go

```diff
@@ -802,6 +802,56 @@ func (c *connection) DeleteProjectInvite(ctx context.Context, id string) error {
 	return nil
 }
 
+func (c *connection) CountOrganizationProjects(ctx context.Context, orgID string) (int, error) {
```

Comment:
Place implementations in same relative location in file as in interface, makes them easier to navigate